persistence/persistence-api/src/main/java/org/cbioportal/persistence/GeneRepository.java

```diff
@@ -31,13 +31,30 @@
 */
 package org.cbioportal.persistence;
 
-import java.util.List;
 import org.cbioportal.model.Gene;
+import org.cbioportal.model.meta.BaseMeta;
+
+import java.util.List;
 
-/**
- *
- * @author jiaojiao
- */
 public interface GeneRepository {
-    List<Gene> getGeneListByHugoSymbols(List<String> hugo_gene_symbol);
+
```

Comment:
I'll add javadocs for all interfaces after I finish implementing APIs in my design list.